subgraph.template.yaml

```diff
@@ -71,6 +71,10 @@ dataSources:
       eventHandlers:
         - event: TransactionRelayed(indexed address,indexed address,indexed address,bytes4,uint8,uint256)
           handler: handleTransactionRelayed
+        - event: Deposited(indexed address,indexed address,uint256)
+          handler: handleDeposited
```

Comment:
maybe i can change name here to `handleDepositedGSN` @gjeanmart ?

Reply:
Yeah, that's a good point. you can rename it like this.

But moving forward to a more and more complex subgraph, I would say, there is several possibilities here to prevent confusions:
1. use a separate mapping file for each smart-contract (factory.ts, wallet.ts, relayhub.ts)
2. use a naming convention handle<contract><event> (handleRelayHubDeposited)

We can either do this now or make an improvement ticket for later.

Reply:
ok i make it an improvement for later, merging it as is -- so i can deploy the same code to mainnet